appinventor/appengine/src/com/google/appinventor/client/wizards/ProjectUploadWizard.java

```diff
@@ -95,7 +99,7 @@ public void onSuccess(UploadResponse uploadResponse) {
                 UserProject userProject = UserProject.valueOf(info);
                 Ode ode = Ode.getInstance();
                 Project uploadedProject = ode.getProjectManager().addProject(userProject);
-                ode.openYoungAndroidProjectInDesigner(uploadedProject);
+                //ode.openYoungAndroidProjectInDesigner(uploadedProject);
```

Comment:
I suspect you commented this line out for your testing, but it needs to be put back!

Reply:
Yeh! Sorry for that, will fix it

Reply:
@jisqyv, Fixed